pocket_coffea/utils/accumulate.py

```diff
@@ -0,0 +1,63 @@
+from coffea.util import load
+
+def initialize_accumulator(o, joint_dataset):
+    '''This function properly initialize the joint accumulator.'''
+    accumulator = {'sum_genweights' : {}, 'sumw' : {}, 'cutflow' : {}, 'variables' : {}}
+    
+    for key in ['sumw', 'cutflow']:
+        for k, d in o[key].items():
+            accumulator[key][k] = {}
+            
+    for key in ['variables']:
+        for k, d in o[key].items():
+            accumulator[key][k] = {joint_dataset : None}
+    
+    return accumulator
+
+def join_accumulator(accumulator, joint_dataset):
+    '''This function joins the total accumulator into the accumulator of a single dataset.'''
+    for key in ['sumw', 'cutflow']:
+        for k, d in accumulator[key].items():
+            accumulator[key][k] = {joint_dataset : sum(d.values())}
+
+    for key in ['sum_genweights']:
```

Comment:
With the new implementation of the `sum_genweights`,  this should be remove from the accumulation process.